testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -54,8 +215,9 @@ public void testTrinoReadingTableCreatedByNativeSpark()
                         "USING ORC\n" +
                         "CLUSTERED BY (a_string)\n" +
                         "INTO 4 BUCKETS\n" +
-                        // Hive requires "original" files of transactional tables to conform to the bucketed tables naming pattern
-                        // We can disable transactions or add another pattern to BackgroundHiveSplitLoader
+                        // By default Spark creates table as "transactional=true", but doesn't conform to Hive transactional format,
```

Comment:
can we also test the default case and assert specific failure?

Reply:
that's orthogonal to file format coverage, which is the focus of this PR. 
this could be a follow up though. it looks like an intentional incomapibility of spark/hive, where spark declares table as transactionbal (because spark itself can modify it?)
i didn't find it important though to follow up immediately